doc/md/privacy.md

```diff
@@ -469,51 +472,49 @@ func DenyMismatchedTenants() privacy.MutationRule {
 		// and it matches the tenant-id of the group above.
-		uid, err := m.Client().User.Query().Where(user.IDIn(users...)).QueryTenant().OnlyID(ctx)
+		userTid, err := m.Client().User.Query().Where(user.IDIn(users...)).QueryTenant().OnlyID(ctx)
 		if err != nil {
-			return privacy.Denyf("querying the tenant-id %w", err)
+			return privacy.Denyf("querying the tenant-id %v", err)
 		}
-		if uid != tid {
-			return privacy.Denyf("mismatch tenant-ids for group/users %d != %d", tid, uid)
+		if userTid != tid {
+			return privacy.Denyf("mismatch tenant-ids for group/users %d != %d", tid, userTid)
 		}
 		// Skip to the next privacy rule (equivalent to return nil).
 		return privacy.Skip
 	})
-	// Evaluate the mutation rule only on group creation.
-	return privacy.OnMutationOperation(rule, ent.OpCreate)
 }
 ```
 
 We add this rule to the `Group` schema and run code-generation.
 
-```go
+```go title="examples/privacytenant/ent/schema/group.go"
 // Policy defines the privacy policy of the Group.
 func (Group) Policy() ent.Policy {
 	return privacy.Policy{
 		Mutation: privacy.MutationPolicy{
-			rule.DenyMismatchedTenants(),
+			privacy.OnMutationOperation(
+				rule.DenyMismatchedTenants(),
+				ent.OpCreate),
 		},
 	}
 }
 ```
 
 Again, we expect the privacy-rules to take effect on the client operations.
 
-```go
+```go title="examples/privacytenant/example_test.go"
 func Do(ctx context.Context, client *ent.Client) error {
     // A continuation of the code-block above.
 
-	// We expect operation to fail, because the DenyMismatchedTenants rule
-	// makes sure the group and the users are connected to the same tenant.
-	_, err = client.Group.Create().SetName("entgo.io").SetTenant(hub).AddUsers(nati).Save(admin)
-	if !errors.Is(err, privacy.Deny) {
-		return fmt.Errorf("expect operatio to fail, since user (nati) is not connected to the same tenant")
+	// Expect operation to fail, because the DenyMismatchedTenants rule makes sure
+	// the group and the users are connected to the same tenant.
+	if _, err = client.Group.Create().SetName("entgo.io").SetTenant(hub).AddUsers(natiLab).Save(adminCtx); !errors.Is(err, privacy.Deny) {
+		return fmt.Errorf("expect operation to fail, since user (natiLab) is not connected to the same tenant")
 	}
-	_, err = client.Group.Create().SetName("entgo.io").SetTenant(hub).AddUsers(nati, a8m).Save(admin)
-	if !errors.Is(err, privacy.Deny) {
-		return fmt.Errorf("expect operatio to fail, since some users (nati) are not connected to the same tenant")
+	if _, err = client.Group.Create().SetName("entgo.io").SetTenant(hub).AddUsers(natiLab, a8mHub).Save(adminCtx); !errors.Is(err, privacy.Deny) {
+		return fmt.Errorf("expect operation to fail, since some users (natiLab) are not connected to the same tenant")
 	}
-	entgo, err := client.Group.Create().SetName("entgo.io").SetTenant(hub).AddUsers(a8m).Save(admin)
+	entgo, err := client.Group.Create().SetName("entgo.io").SetTenant(hub).AddUsers(a8mHub).Save(adminCtx)
```

Comment:
Please revert to the previous version, or explain why this one is preferred.

These lines just feel too long for me.